README.md

```diff
@@ -36,7 +36,10 @@ More Info on released versions: https://www.modelica.org/documents
 ## Contribution
 1. If you find an error and are not certain that you can correct it, first check that it is not already reported and then open an [issue](https://github.com/modelica/ModelicaSpecification/issues) describing it in detail - focusing on why it should be changed.
 2. If you are confident that you can correct the issue, fork this repository and create a pull-request and in the pull-request explain the issue and the correction; you will also have to sign a CLA.
-3. Significant extensions are handled as Modelica Change Proposals. (Template to follow.) This can start as a simple description of the proposed extension. It will then be worked on to have a rationale explaining how the change help users, and demonstrating that it can be implemented efficiently; and finally a pull-request with the changes.
+3. Significant extensions are handled as Modelica Change Proposals (MCPs, Template to follow.) This can start as a simple description of the proposed extension. It will then be worked on to have a rationale explaining how the change help users, and demonstrating that it can be implemented efficiently; and finally a pull-request with the changes.
```

Comment:
I agree that the previous _(Template to follow.)_ wasn't a very good sentence, but I think that the mention of templates is a rest from the time before we moved to GitHub and text-based document source format:
```suggestion
3. Significant extensions are handled as Modelica Change Proposals (MCPs). This can start as a simple description of the proposed extension. It will then be worked on to have a rationale explaining how the change help users, and demonstrating that it can be implemented efficiently; and finally a pull-request with the changes.
```

Reply:
The template is actually found in the MCP-directory, but that is linked below so I don't think we need it here.